tests/trezor/trezor_tests.cpp

```diff
@@ -625,24 +782,31 @@ gen_trezor_base::gen_trezor_base(const gen_trezor_base &other):
     m_bob_account(other.m_bob_account), m_alice_account(other.m_alice_account), m_eve_account(other.m_eve_account),
     m_hard_forks(other.m_hard_forks), m_trezor(other.m_trezor), m_rct_config(other.m_rct_config), m_top_hard_fork(other.m_top_hard_fork),
     m_heavy_tests(other.m_heavy_tests), m_test_get_tx_key(other.m_test_get_tx_key), m_live_refresh_enabled(other.m_live_refresh_enabled),
-    m_network_type(other.m_network_type), m_daemon(other.m_daemon)
+    m_network_type(other.m_network_type), m_daemon(other.m_daemon), m_alice2_account(other.m_alice2_account),
+    m_trezor_path(other.m_trezor_path), m_trezor_pin(other.m_trezor_pin), m_trezor_passphrase(other.m_trezor_passphrase),
+    m_trezor_use_passphrase(other.m_trezor_use_passphrase), m_trezor_use_alice2(other.m_trezor_use_alice2),
+    m_gen_heavy_test_set(other.m_gen_heavy_test_set)
```

Comment:
`m_head`, `m_events`, `m_mining_enabled`, `m_mining_timeout` and perhaps some other fields are missing in this copy constructor.